ethereum_history_api/circuits/lib/src/misc/arrays_test.nr

```diff
@@ -70,3 +70,23 @@ fn test_array_not_equals() {
     let other_array: [Field; 3] = [1, 2, 4];
     assert(array_equals(array, other_array), "Arrays not equal");
 }
+
+#[test]
+fn test_resize() {
```

Comment:
New test cases proposal: 
- test_resize_empty()
- test_resize_big_padding()

Reply:
What will the second test test?
It feels to me - that the size of the padding doesn't matter. Unless you want to test some buffer overflows or out of memory errors and pad stuff to gigabytes

Reply:
Added the first test